docs/uml-class-diagrams/middleware02/test/TrackDataCalculatorTests/TrackDataCalculatorTests.puml

```diff
@@ -2,7 +2,9 @@
 
 package edu.oakland.test.middleware02 {
     class TrackDataCalculatorTests {
-
+      + void calculateSpeedTest()
+      + void calculateDirectionTest()
+      + void calculateRootMeanSquaredTest()
```

Comment:
You don't have to take this suggestion if you don't want to; you're free to merge with or without it, but I would maybe make this as a private method that you place inside of the other two test methods in the event you do get around to making this with Root Mean Squares. By doing so, it just means you don't need to expose the RMS method publicly outside of the TrackDataCalculator, but if you think it will be easier to build if you can test that part separately from the rest, by all means go ahead!
```suggestion
      - void calculateRootMeanSquaredTest()
```